src/Components/Home/Home.css

```diff
@@ -11,12 +11,32 @@ a {
 
 .p-chip:hover {
   transform: scale(1.1);
-  box-shadow: rgba(255, 255, 255, 0.2) 0px 0px 0px 1px inset,
-    rgba(0, 0, 0, 0.9) 0px 0px 0px 1px;
+  background: linear-gradient(145deg, #c8cbcf, #eef2f6);
+  box-shadow: 8px 8px 15px #bdc0c4, -8px -8px 15px #ffffff;
+}
+
+.p-chip .p-avatar img {
```

Comment:
Is this custom CSS not something that can be achieved with prime react classes? (lines 18 - 30)

Reply:
prime-react is an UI component library it has no small CSS classes to work with in this case, the prime-flex has some but those are related to layout. Please  correct me if I'm wrong , have not prior experience using prime-react.

Reply:
Just wanted to check, I’m not well-versed in prime react either, I just know we’re trying to avoid custom css as much as possible.